migrator/migrator-servlet/src/main/webapp/index.jsp

```diff
@@ -8,7 +8,10 @@
 <%
   final String E164_NUMBERS_LINK = "https://support.twilio.com/hc/en-us/articles/223183008-Formatting-International-Phone-Numbers";
   final String COUNTRY_CODE_LINK = "https://countrycode.org/";
-  final String DOCUMENTATION_LINK = "./"; // TODO: use README documentation link when uploaded
+  // TODO: use documentation link from base repository when forked repository has been merged in
+  final String DOCUMENTATION_LINK = "https://github.com/TomiwaOke/libphonenumber/tree/master/migrator/README.md";
+  final String BUGANIZER_LINK = "https://b.corp.google.com/issues/new?component=192347&template=829703";
```

Comment:
ISSUE_TRACKER_LINK (which is the external name of Buganizer)

Reply:
Also, that link probably doesn't work outside of Google. Is there a public issue tracker set up for this?